pkg/apis/scheduling/v1beta1/types.go

```diff
@@ -301,11 +301,36 @@ type QueueStatus struct {
 
 // CluterSpec represents the template of Cluster
 type Cluster struct {
-	Name string              `json:"name,omitempty" protobuf:"bytes,1,opt,name=name"`
-	Weight int32             `json:"weight,omitempty" protobuf:"bytes,2,opt,name=weight"`
+	Name     string          `json:"name,omitempty" protobuf:"bytes,1,opt,name=name"`
+	Weight   int32           `json:"weight,omitempty" protobuf:"bytes,2,opt,name=weight"`
 	Capacity v1.ResourceList `json:"capacity,omitempty" protobuf:"bytes,3,opt,name=capacity"`
 }
 
+// Affinity is a group of affinity scheduling rules.
+type Affinity struct {
```

Comment:
prefer to have two gorups: `Affinity` & `AntiAffinity`.

Reply:
```go
// k8s.io/api/core/v1/types.go
type Affinity struct {
	NodeAffinity *NodeAffinity `json:"nodeAffinity,omitempty" protobuf:"bytes,1,opt,name=nodeAffinity"`
	PodAffinity *PodAffinity `json:"podAffinity,omitempty" protobuf:"bytes,2,opt,name=podAffinity"`
	PodAntiAffinity *PodAntiAffinity `json:"podAntiAffinity,omitempty" protobuf:"bytes,3,opt,name=podAntiAffinity"`
}
```

```yaml
apiVersion: scheduling.volcano.sh/v1beta1
   kind: Queue
   metadata:
     name: default
     spec:
       reclaimable: true
       weight: 1
       affinity:
         nodeGroupAffinity:
         nodeGroupAntiAffinity:
         queueAffinity:
         queueAntiAffinity:
```

Reply:
That's ok to me. It's better get more comments from other maintainers :)